packages/shared/lib/network.ts

```diff
@@ -29,13 +29,19 @@ export const CHRYSALIS_DEVNET_BECH32_HRP = 'atoi'
  *
  * @returns {NetworkConfig}
  */
-export const getOfficialNetworkConfig = (type: NetworkType): NetworkConfig => ({
-    network: getOfficialNetwork(type),
-    nodes: getOfficialNodes(type),
-    automaticNodeSelection: true,
-    includeOfficialNodes: true,
-    localPow: true,
-})
+export const getOfficialNetworkConfig = (type: NetworkType): NetworkConfig => {
+    const officialNodes = getOfficialNodes(type)
+    const randIdx = Math.floor(Math.random() * officialNodes.length)
```

Comment:
There is a [`pick`](https://github.com/iotaledger/firefly/blob/feat/staking/packages/shared/lib/utils.ts#L351) function in `lib/utils.ts` that will randomly select the element of an array - we can de-dup some code a bit and substitute it here.

Reply:
@maxwellmattryan that returns the specific element of the array, where I want to modify a specific element in an array and keep the rest of the array too.

Although there is this exact code further down that I can extract into its own function.

Reply:
Made it a little more DRY 😊